docs/contribute/Scenarioo-User-Documentation-and-Webpage-with-Jekyll.md

```diff
@@ -1,51 +1,45 @@
 # Scenarioo Webpage and Documentation
 
 
-**TODO: this info here is probably not 100% accurate, ask @bruderol, this needs to be corrected!**
+## Web Page
 
-We use Jekyll to generate our webpage and documentation on the github pages.
+The scnearioo.org webpage is maintained in repository 
+https://github.com/scenarioo/scenarioo.github.io
 
-What is special: 
-We do not directly maintain the docu files in the gh-pages branch. We edit our pages in the 'docu' directory on every branch and finaly release from the docu directory on master branch to gh-pages.
+The web page can be changed in that repo by simply do a PR to master.
 
-## Jekyll Setup for generating Webpage and Documentation
+See README.md in that repo for further detailed information.
 
-You need to install Jekyll and all prerequisites on your development environment machine, as explained here under https://help.github.com/articles/using-jekyll-with-pages/.
+Changes for further releases should not yet been made on master but prepared on special release branches, to be merged on the date of release.
 
-This was what I did on the scenarioo dev VM to get it installed:
+## Documentation
 
- * Install Ruby 2.x:
-   See also http://rvm.io/ for how to install ruby version manager (RVM) to get newest ruby version, I did the following steps on the VM:
-    * `sudo apt-get update`
-    * `gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3`
-    * `sudo apt-get install curl`
-    * `\curl -sSL https://get.rvm.io | bash -s stable`
-    * `source /home/scenarioo/.rvm/scripts/rvm`
-    * `rvm get stable`
-    * `rvm install ruby2.2.1`
- * Install Jekyll:
-    * `gem install bundler`
-    * call the following command inside the 'scenarioo.github.io' directory (where Gemfile is checked in): 
-      `bundle install`
+We do not use Wikis!
 
-## How to maintain/edit the docu
+All our repos have a docs folder with a markdown file documentation using gitbook tooling.
 
-Simply change the html and markdown files in the repository 'scenarioo.github.io'.
+The major scenarioo documentation is in major scenarioo repository here:
+https://github.com/scenarioo/scenarioo/tree/develop/docs
 
-**Attention**: If you work for documentation of a future release, then do not work on the main branch but change the docu on the appropriate branch!
+###  How to Publish Documentation
 
-## How to generate and review docu
+* Make sure the version of scenarioo/scenarioo you want to release docu for is checked out (usually `develop`, `master` or a release branch).
 
-you can browse the webpage locally, by calling the following commands inside directory 'scenarioo.github.io':
- * `jekyll serve`
- * browse to localhost:4000
+* Is it docu for a new release version of scenarioo? If not, continue with next step, otherwise do the following first ... 
```

Comment:
Simplify this sentence:

`If you publish documentation for a release, do the following, otherwise skip to the next step.`

Reply:
As I understood you yesterday, the develop branch is now automatically published. So why is there any documentation that is not specific to making a release?

Reply:
I first wanted to document what we have to do - in case automation will not work - currently the automation is still shaky and we might have to do it manually sometimes. But I will adjust that once it works.

Reply:
documenteimproved docu accordingly to reflect the current state of it